src/workloads/execution/ExpressiveQueries.yml

```diff
@@ -0,0 +1,207 @@
+        zip: {^RandomString: {length: 6}},
+        location: {position: {type: "Point", coordinates: [
+          {^RandomDouble: {min: -118.668469, max: -82.062023}},
+          {^RandomDouble: {min: 32.924436, max: 41.679986}}
+        ]}}
+      }
+      policies: {^Array: {number: {^RandomInt: {min: 3, max: 6}}, of: {^Choose: {from: [
+        {
+          policyType: "auto",
+          policyNum: {^RandomString: {length: 30}},
+          nextRenewalDt: {^RandomDate: {min: "2017-01-01", max: "2017-12-31"}},
+          year: {^RandomInt: {min: 1980, max: 2017}},
+          value: {^RandomInt: {min: 500, max: 500000}},
+          model: {^RandomString: {length: 30}},
+        },
+        {
+          policyType: "home",
+          policyNum: {^RandomString: {length: 30}},
+          nextRenewalDt: {^RandomDate: {min: "2017-01-01", max: "2017-12-31"}},
+          year: {^RandomInt: {min: 1970, max: 2017}},
+          value: {^RandomInt: {min: 500, max: 10000000}},
+          address: {
+            number: {^RandomInt: {min: 1, max: 9999}},
+            street: {^RandomString: {length: {^RandomInt: {min: 30, max: 60}}}},
+            city: {^RandomString: {length: {^RandomInt: {min: 5, max: 10}}}},
+            state: {^Choose: {from: ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE", "DC", "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS", "KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY", "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC", "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY"]}},
+            zip: {^RandomString: {length: 6}},
+            location: {position: {type: "Point", coordinates: [
+              {^RandomDouble: {min: -118.668469, max: -82.062023}},
+              {^RandomDouble: {min: 32.924436, max: 41.679986}}
+            ]}}
+          }
+        },
+        {
+          policyType: "life",
+          policyNum: {^RandomString: {length: 30}},
+          nextRenewalDt: {^RandomDate: {min: "2017-01-01", max: "2017-12-31"}},
+          insured_person: {
+            firstname: {^RandomString: {length: {^RandomInt: {min: 3, max: 10}}}},
+            lastname: {^RandomString: {length: {^RandomInt: {min: 5, max: 10}}}},
+            yob: {^RandomDate: {min: "1930-01-01", max: "2016-12-31"}},
+            smoking: {^RandomInt: {min: 0, max: 1}}
+          }
+        }
+      ]}}}}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: ManageDatabase
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        fsync: 1
+  - *Nop
+  - *Nop
+  - Repeat: 5
```

Comment:
Why do we need Repeat: 5 here? Wouldn't 1 be enough? [I guess CreateIndexPhase do it intentionally several times to get accurate timing for index creation  - my understanding that we don't care about that here]

Reply:
1 should be fine. Also, the loader is perfectly happy to add an index for you. ~Do you need the index build in a different phase than the loader?~ Consider adding a comment repeating that you need to add the comment after running the queries without the index